tests/testthat/helper-functions.R

```diff
@@ -419,11 +419,12 @@ find_test_path <- function(dir = ".") {
 
 runtime_evaluate <- function(script=NULL){
   
-  eval.result<- remDr$Runtime$evaluate(script,returnByValue = TRUE)
+  eval.result<- remDr$Runtime$evaluate(script,returnByValue = TRUE)$result$value
```

Comment:
great, but please delete `eval.result<-` because the result is immediately returned (and `eval.result` is never used)
also please undo addition of empty lines

Reply:
thank you for pointing that out. I was under impression that we either need to save to a variable or use return statement explicitly for the function to return.

Reply:
not true
result of last statement in function is returned, without using return(), without using `<-`.
return() is only necessary if you want to return early.

Reply:
changed the code to directly return result of Runtime$evaluate API without saving it in variable